cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -491,3 +587,48 @@ func (r *KubeArchiveConfigReconciler) removeFilters(ctx context.Context, kaconfi
 
 	return nil
 }
+
+func (r *KubeArchiveConfigReconciler) reconcileNamespace(ctx context.Context, kaconfig *kubearchivev1alpha1.KubeArchiveConfig) (*corev1.Namespace, error) {
+	log := log.FromContext(ctx)
+
+	log.Info("in reconcileNamespace")
+
+	ns := &corev1.Namespace{}
+	err := r.Get(ctx, types.NamespacedName{Name: kaconfig.Namespace}, ns)
+	if err == nil {
+		ns.ObjectMeta.Labels[ApiServerSourceLabelName] = ApiServerSourceLabelValue
+		err = r.Update(ctx, ns)
+		if err != nil {
+			log.Error(err, "Failed to update Namespace "+kaconfig.Namespace)
+			return ns, err
+		}
+	} else {
+		log.Error(err, "Failed to get Namespace "+kaconfig.Namespace)
+		return ns, err
+	}
+
+	return ns, nil
+}
+
+func (r *KubeArchiveConfigReconciler) removeNamespaceLabel(ctx context.Context, kaconfig *kubearchivev1alpha1.KubeArchiveConfig) error {
+	log := log.FromContext(ctx)
+
+	log.Info("in removeNamespaceLabel")
+
+	ns := &corev1.Namespace{}
+	err := r.Get(ctx, types.NamespacedName{Name: kaconfig.Namespace}, ns)
+	if err != nil {
+		log.Error(err, "Unable to get Namespace "+kaconfig.Namespace)
```

Comment:
All other error messages of this type use "Failed to..." in the description. You should do the same here to keep the code consistent.

```suggestion
		log.Error(err, "Failed to get Namespace "+kaconfig.Namespace)
```